tools/update-client

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# This script creates a new release of the Hypothesis browser extension
+# using the latest version of the client available on npm.
+
+set -eu
+
+LATEST_CLIENT_VERSION=$(npm view hypothesis version)
+
+# Check that working tree is clean
+if ! git diff --quiet HEAD ; then
+  echo "There are uncommitted changes in the working tree or index."
+  exit 1
+fi
+
+# Update checkout
+git checkout master
```

Comment:
Hmm. I still think it'd be better to check whether they are on master and bail if they aren't, rather than quietly changing the user's branch on them which they may not expect or notice.